packages/nc-gui/components/smartsheet/Form.vue

```diff
@@ -777,13 +777,13 @@ watch(view, (nextView) => {
 
           <!-- Show this message -->
           <div class="text-gray-500 text-bold">{{ $t('msg.info.showMessage') }}:</div>
-          <a-textarea
-            v-model:value="formViewData.success_msg"
-            :rows="3"
-            hide-details
-            class="nc-form-after-submit-msg"
-            data-testid="nc-form-after-submit-msg"
-            @change="updateView"
+         <a-textarea
+        v-model:value="formViewData.success_msg"
+        :rows="3"
+         hide-details
+          class="nc-form-after-submit-msg"
+          data-testid="nc-form-after-submit-msg"
+          @change="updateView"
```

Comment:
Unexpected indentation. Please run lint or just replace the old one back.

Reply:
Okay will do that.